ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -55,6 +56,9 @@ import TreegraphLayout from './TreegraphLayout.js';
 import { TreegraphSeriesLevelOptions } from './TreegraphSeriesOptions.js';
 import TreegraphSeriesDefaults from './TreegraphSeriesDefaults.js';
 import TreemapPoint from '../Treemap/TreemapPoint.js';
+import SVGElement from '../../Core/Renderer/SVG/SVGElement.js';
+import TextPath from '../../Extensions/TextPath.js';
+TextPath.compose(SVGElement);
```

Comment:
This compose call should run automatically sole in the masters containing treegraph. Treegraph also needs a static class function `compose`, if this is essential for Treegraph.

Reply:
Or are you saying alter `treegraph.src.ts` to call this static compose function?

Reply:
This is necessary for alle the series here. Right now this extension is not an extension. It is not compatible with async ES imports and shaked bundling and is bundled everywhere.

Things to consider:

* If these series require the text path feature, they have to provide a compose function taking `SVGElement` for modification. The question then is, why do we have it as an extension?

* If it is optional, a call in `textpath` master is enough and the series do not need their own `compose`.

* The related masters need to call the compose function(s) in every case. They are our official bundles.

* If you want a truly extension, the `textpath` master needs to be the only one composing it. The `textpath` master can be imported by every other master that should contain the composition.

Reply:
Okay, maybe we should discuss this offline, but here are the facts/requirements:
* TextPath is an extension to the SVGElement, allowing users to add text paths to text elements.
* So save weight, we don't want it in the main bundle.
* We provide a module bundle, `module/textpath.js`, so that users can use it in custom rendering.
* Some series types use text paths by default. For those series types, we want to embed the TextPath capabilities in the series type bundle.

Reply:
The compose in all series should still go into master files. It is not expected to have changes to SVGElement just by importing a series type. If this happens async, users have no chance of control when what gets modified. So remove the compose call from the series class files and add an import of the textpath master to the series masters regarding affected series types.

Reply:
> users have no chance of control when what gets modified

The thing is that text paths just don't work without the extension. The data labels will not render correctly. So I'm not convinced that this should go in the masters. It's not a _modification_ per se of the SVGElement, it is the addition of a feature. I would imagine that if an implementer loads the Treegraph module using ESM and the data labels are all over the place, that would be surprising. I wouldn't think of loading the extra TextPath module explicitly.

Reply:
I see, so it is a must have.

Reply:
Ok so I treegraph series now calls `TextPath.compose` as it was originally

I tried to revert the commit but somehow ended up merging over it when pulling from master, hopefully nothing weird happened to the branch :grimacing: